elytron/ELY-1712-enhanced-audit-logging.adoc

```diff
@@ -0,0 +1,73 @@
+= [WFCORE-4394] Enhanced Audit Logging - RFC support and reliability/speed customization
+:author:            Justin Michael Cook
+:email:             jucook@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Currently, Audit Logging in Elytron does not have the same capabilities as Legacy Security. This task is to enhance Elytron Audit Logging to have the same capabilities as Legacy Security Audit Logging.
```

Comment:
@justinmcook Just one last change. Can you please edit the overview so it reflects current state of this analysis?

Reply:
Edited the overview